mcodingbot/plugins/python.py

```diff
@@ -29,9 +30,10 @@ def get_dismiss_button(id: hikari.Snowflake) -> hikari.api.ActionRowBuilder:
     return action_row
 
 
-def get_pep_link(pep_number: int, *, hide_embed: bool) -> str:
-    url = f"https://peps.python.org/pep-{pep_number:04}/"
-    return f"<{url}>" if hide_embed else url
+@plugin.include
+@tasks.loop(hours=1)
```

Comment:
Honestly PEPs are not updated every hour, it can be a day-checking imo

Reply:
If we are doing every day a cronjob would definitely be better. That way its not dependent on bot start time and I dont need to import datetime.

Reply:
> If we are doing every day a cronjob task would definitely be better. That way its not dependent on bot start time and I dont need to import datetime.

Sure!